source/docs/embark_commands.md

```diff
@@ -66,6 +66,10 @@ Option | Description
 
 Runs Tests. If `file` is not specified then it will run all the tests inside the `test/` directory.
 
+Option | Description
+--- | ---
+`--coverage` | generate solidity coverage report
```

Comment:
Might I ask if you would be so kind as to add the --node option too :)

Reply:
On it!